test/functions/default-arguments/default-argument-inout.chpl

```diff
@@ -0,0 +1,3 @@
+proc foo(inout x = 7) { x = 17; writeln(x); }
+foo();
+
```

Comment:
I don't like that these tests have an extra line at the end of them, but I'm willing to live with it.  That said, I'm going to wait until Monday to merge so if you get motivated... :p

Reply:
Motivation found.  Here y'go.